ocaml/xapi/xapi_pool.ml

```diff
@@ -1059,6 +1059,7 @@ let rec create_or_get_host_on_master __context rpc session_id (host_ref, host) :
           ~console_idle_timeout:host.API.host_console_idle_timeout
           ~ssh_auto_mode:host.API.host_ssh_auto_mode
           ~secure_boot:host.API.host_secure_boot
+          ~software_version:(Xapi_globs.software_version ())
```

Comment:
No, you should use `host.API.host_software_version`. This is the field as it is on the joining host, which we want to copy to the pool.